index.html

```diff
@@ -109,9 +109,9 @@ <h3>
 
         <!-- Sponsor links -->
         <h3>Sponsored by:</h3>
-        <div class="sponsor">
-            <a href="https://trackjs.com/" class="sponsor-a" target="_blank">
-                <img src="img/trackjs_logo.png" alt="TrackJS Logo" class="img-100">
+        <div class="alignToCenter">
+            <a href="https://trackjs.com/" class="alignToCenter" target="_blank">
+                <img src="img/trackjs_logo.png" alt="TrackJS Logo" class="alignToCenter">   
```

Comment:
Let's try this again, we should be able to retain the `img-100` class on this `<img>` tag. Let's just modify the `sponsor` and `sponsor-a` classes rather than creating a new one.

Also, we shouldn't need to use this `alignToCenter` class at all 3 levels if done correctly